profile/README.md

```diff
@@ -12,7 +12,7 @@ Included in the TrustyAI core library are several tools, which include:
 - Fairness metrics
 - Drift metrics
 - Language performance metrics
-- KServe and ModelMesh integration
+- KServe (ODH and side-car) and ModelMesh integration
```

Comment:
I would add a couple of links here, "ODH" is not that clear for user most probably and same for "side-car"

Reply:
I've added a link to the TrustyAI docs for ODH and to KServe docs for the side-car.

Reply:
I've rewritten that portion of the text to try to make the distinction clearer.